replicated_kv_store/raft_node_test.go

```diff
@@ -19,9 +19,11 @@ func TestInitializeNode(t *testing.T) {
 	 * the replica ID
 	 */
 	var no_of_replicas, replica_id int32
+	var replica_key_value_port string
 	no_of_replicas = 5
 	replica_id = 1
-	node := InitializeNode(no_of_replicas, replica_id)
+	replica_key_value_port = ":8081"
```

Comment:
I think an `if` condition could be added to this test case to check the value of the key value service port number set in the raft node struct, essentially the member `replicated_keyvaluestore`, whether it is the same as the one supplied to the function. While it does seem trivial, it is the expected behavior so it must be checked in the test case.